test/module/irohad/ametsuchi/flat_file_test.cpp

```diff
@@ -97,8 +96,8 @@ TEST_F(BlStore_Test, BlockStoreWhenAbsentFolder) {
   log_->info(
       "----------| Check that folder is absent => create => "
       "make storage => remove storage |----------");
-  std::string target_path = "/tmp/bump";
-  rmdir(target_path.c_str());
+  auto target_path = fs::temp_directory_path() / fs::unique_path();
+  fs::remove(target_path.c_str());
   {
     auto store = FlatFile::create(block_store_path);
```

Comment:
I don't get it completely, but I guess block_store_path is expected to be a file in target_path directory.

Reply:
I didn't get the idea of the test either, just replaced with the boost functions.

Original values are:
`block_store_path` is "/tmp/**d**ump"
`target_path` is "/tmp/**b**ump"

Reply:
Seems there's an old-living bug here